### PR TITLE
Right click doc update

### PR DESCRIPTION
**Pull request by BryceV**
Added information about new "actionType" and "func" in right-click menus.

### CODE REVIEW

File: settings.html [hunk 1]
```diff
@@ -3492,6 +3491,35 @@ <h2 id="right-click">
               name if the name field is missing
             </td>
           </tr>
+          <tr>
+            <td id="right-click-key">
```
Comment (awick) [inline settings.html:3495]: this should be actionType

File: settings.html [hunk 2]
```diff
@@ -3492,6 +3491,35 @@ <h2 id="right-click">
...
+              actionType=
```
Comment (awick) [inline settings.html:3496]: no =

File: settings.html [hunk 3]
```diff
@@ -3492,6 +3491,35 @@ <h2 id="right-click">
...
+            </td>
+            <td>
+              Needs a url. Supported actionTypes:
+              <ul class="mb-0">
+                <li>fetch: Information will be fetched and displayed in the rightclick menu for 5 seconds</li>
+              </ul>
+            </td>
+          </tr>
+          <tr>
+            <td id="right-click-key">
```
Comment (awick) [inline settings.html:3509]: should be "func"

File: settings.html [hunk 4]
```diff
@@ -3492,6 +3491,35 @@ <h2 id="right-click">
...
+              func=
```
Comment (awick) [inline settings.html:3510]: no =

File: settings.html [hunk 5]
```diff
@@ -3492,6 +3491,35 @@ <h2 id="right-click">
...
+              <span class="fa fa-link small copy-link cursor-copy"
+                onclick="copyLink(this, 'settings')">
+              </span>
+            </td>
+            <td>
+              A function body that will feed into new Function() and returns a right click action formatted as an object.
```
Comment (awick) [inline settings.html:3516]: A javascript function body

File: settings.html (hunk 1 above)
Comment (BryceV) [reply settings.html]: fixed

File: settings.html (hunk 2 above)
Comment (BryceV) [reply settings.html]: fixed

File: settings.html (hunk 3 above)
Comment (BryceV) [reply settings.html]: fixed

File: settings.html (hunk 4 above)
Comment (BryceV) [reply settings.html]: fixed

File: settings.html (hunk 5 above)
Comment (BryceV) [reply settings.html]: added

**Pull request merged**